can/io/asc.py

```diff
@@ -98,14 +127,26 @@ def _extract_header(self) -> None:
                 self.timestamps_format = timestamp_format or "absolute"
                 continue
 
+            if asc_version_match:
+                asc_version = asc_version_match.group("version")
+                self.asc_version = Version(asc_version)
+                continue
+
             if comment_match:
                 continue
 
             if events_match:
                 self.internal_events_logged = events_match.group("no_events") is None
-                break
+                continue
 
-            break
+            if trigger_match:
```

Comment:
I think the files can contain multiple triggers. Parsing it as part of the header is probably not sufficient.